pages/success.tsx

```diff
@@ -56,7 +58,7 @@ export default function Success(props) {
     return encodeURIComponent(event.value);
   }
 
-  return (
+  return isReady && (
```

Comment:
Codacy has a fix for the issue: [Insert `(⏎····`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  return (
```